config/database.yml

```diff
@@ -1,25 +0,0 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
```

Comment:
I created `database.yml.sample` file, which can be safely stored in a repository.